From efbb9ed8d55f6c3826b71b4025514dfc0aad34ae Mon Sep 17 00:00:00 2001
From: Nirbheek Chauhan <nirbheek@centricular.com>
Date: Thu, 27 Feb 2020 12:56:13 +0530
Subject: [PATCH 2/2] meson: Do not build tests on iOS

The tests use `system()` which is explicitly marked as not available
on iOS:

```
 ../tests/test-is-public.c:200:11: error: 'system' is unavailable: not available on iOS
                        return system(cmd) != 0;
                               ^
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.2.sdk/usr/include/stdlib.h:190:6: note: 'system' has been explicitly marked unavailable here
int      system(const char *) __DARWIN_ALIAS_C(system);
         ^
```
---
 meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/meson.build b/meson.build
index 773e6af..eb17810 100644
--- a/meson.build
+++ b/meson.build
@@ -144,5 +144,3 @@ endif
 subdir('include')
 subdir('src')
 subdir('tools')
-subdir('tests')
-subdir('fuzz')
--
2.21.1 (Apple Git-122.3)

